endoreg_db/models/media/pdf/raw_pdf.py

```
@@ -5,35 +5,42 @@
+from django.db import models
+from numpy import isin  # Import Django File
+
 from endoreg_db.utils.file_operations import get_uuid_filename
-from typing import TYPE_CHECKING, Optional
-# Use the specific paths from the centralized paths module
-from ...utils import PDF_DIR, STORAGE_DIR
 from endoreg_db.utils.hashs import get_pdf_hash
+from endoreg_db.utils.paths import PDF_DIR, RAW_PDF_DIR
+
+# Use the specific paths from the centralized paths module
+from ...utils import PDF_DIR
```

Ruff (0.14.3)

Redefinition of unused `PDF_DIR` from line 18

(F811)